src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -52,9 +54,7 @@ const PermissionIdentities: FC = () => {
   const { hasAccessManagementTLS } = useSupportedFeatures();
 
   useEffect(() => {
-    const validIdentities = new Set(
-      identities.map((identity) => identity.name),
-    );
+    const validIdentities = new Set(identities.map((identity) => identity.id));
```

Comment:
```suggestion
    const validIdentityIds = new Set(identities.map((identity) => identity.id));
```
I think we should be precise on what this is containing -- not the identity objects, but ids.